ansible/site.yml

```diff
@@ -1,5 +1,34 @@
 ---
+
+# Note: OS specific variables are included in
+# the order of the hierarchy of their fork...
+# The "lower" fork wins
+# Debian <
+#  Ubuntu <
+#   Raspbian
+#   Armian
+
 - hosts: all
+  pre_tasks:
+    - name: Include OS family variables
+      include_vars: "group_vars/{{ ansible_os_family|lower }}"
+    - name: Include Distribution variables
+      include_vars: "group_vars/{{ ansible_distribution|lower }}"
```

Comment:
Importing `group_vars/{{ ansible_distribution}}` seems to override all variables in `group_vars/{{ ansible_os_family}}` so the latter seems redundant and can be removed.